cg/cli/backup.py

```diff
@@ -34,6 +34,51 @@ def backup(context: CGConfig):
     pass
 
 
+@backup.command("flow-cells")
+@DRY_RUN
+@click.pass_obj
+def backup_flow_cells(context: CGConfig, dry_run: bool):
+    """Backup flow cells."""
+    pdc_api = PdcAPI(binary_path=context.pdc.binary_path, dry_run=dry_run)
+    if pdc_api.is_dcms_running():
+        exit(0)
```

Comment:
```suggestion
    if pdc_api.is_dcms_running():
        exit(0)
```

I think we should this check in the `PdcAPI` and remove it from the CLI

Reply:
Fixed